doc/scaleout/Multi_Node_JBOD_Setup.rst

```diff
@@ -226,7 +226,7 @@ Step 4 - Create LVM configuration for the remaining OS partitions using md1 RAID
     
 5. If you have Mellanox HCAs on your servers, please proceed to the next step. If not, proceed to step 8.
 
-6. Install Mellanox OFED from http://linux.mellanox.com/public/repo/mlnx_ofed/4.7-3.2.9.0/rhel7.7/x86_64/MLNX_LIBS/. 
+6. Install Mellanox OFED from http://linux.mellanox.com/public/repo/mlnx_ofed/4.9-0.1.7.0/rhel7.8/
```

Comment:
@Saumya-Sunder May be we should use `http://linux.mellanox.com/public/repo/mlnx_ofed/4.9-0.1.7.0/rhel7.8/x86_64/MLNX_LIBS/`  just to be exact same for CentOS 7.7.1908 URL.

Reply:
Would the contributors have a choice here or do we need it to be exact x86_64?

Reply:
It will always `x86_64` .   - I was just making it same as previous 4.7 version link. 
Old - http://linux.mellanox.com/public/repo/mlnx_ofed/4.7-3.2.9.0/rhel7.7/x86_64/MLNX_LIBS/
New -http://linux.mellanox.com/public/repo/mlnx_ofed/4.9-0.1.7.0/rhel7.8/
Hence suggested to use -
http://linux.mellanox.com/public/repo/mlnx_ofed/4.9-0.1.7.0/rhel7.8/x86_64/MLNX_LIBS/

Reply:
done, @shailesh-vaidya @johnbent, please approve the PR